### ISSUE TITLE
[css-contain] SIze contain shouldn't apply to tables

### ISSUE THREAD

**Issue by MReschenberg** [w3c/csswg-drafts#2746]
[The section on size containment](https://drafts.csswg.org/css-contain/#containment-size) says size containment has no effect on internal table elements, however it seems like it also shouldn't apply to tables (which aren't internal table elements). 

cc: @dbaron

**Comment by dbaron** [comment]
Yeah, I don't think it makes sense for it to apply to tables, because tables don't have a sensible way of being smaller than their contents.  That's never been defined and it's not particularly clear what their contents would *do* (i.e., how wide to make each column).

**Comment by frivoal** [comment]
Oops, that sounds right, I don't quite see how it would work either. Sorry I phrased that wrong.

Ping: @tabatkins @eaenet @mrego. I suspect you all agree, but just wanted to check before updating the spec.

**Comment by tabatkins** [comment]
Yeah, sounds reasonable

**Comment by frivoal** [comment]
Agenda+ to confirm the proposed change.

**Comment by jonjohnjohnson** [comment]
I can recall times when I've used tables with `table-layout: fixed` and `width` not set to `auto`, where I'd be surprised to find out `contain` has no effect. Would this then force me to use a bunch of non-semantic divs with table structure `display` values? I'm primarily thinking of highly responsive layouts where I'm simulating element query behaviors, guessing I'd gain decent perf from using `contain`.

**Comment by frivoal** [comment]
With table-layout:fixed, you can set the width of the table to less than its min-content width, so that sounds tempting, but:
* you still cannot set the height to less than the min-content height
* what happens when the width is less than the min-content is not fully interoperable

So, I think we still need to exclude tables entirely, including when table-layout is fixed.

**Comment by tabatkins** [comment]
> Would this then force me to use a bunch of non-semantic divs with table structure display values?

Note that when we say "tables" here, we're referring to anything using `display:table`. So switching to `<div style="display:table">` won't change anything.

**Comment by jonjohnjohnson** [comment]
> Note that when we say "tables" here, we're referring to anything using display:table. 

Ya, I figured. :)

**Comment by mrego** [comment]
Sounds good to me too.

**Comment by mrego** [comment]
The test has been modified in https://github.com/web-platform-tests/wpt/pull/11502.
